docs/en/seatunnel-engine/deployment.md

```diff
@@ -144,6 +144,42 @@ TCP is our suggest way in a standalone SeaTunnel Engine cluster.
 
 On the other hand, Hazelcast provides some other service discovery methods. For details, please refer to [hazelcast network](https://docs.hazelcast.com/imdg/4.1/clusters/setting-up-clusters)
 
+### 5.3 Map
+
+MapStores connect to an external data store only when they are configured on a map. This topic explains how to configure a map with a MapStore. For details, please refer to [hazelcast map](https://docs.hazelcast.com/imdg/4.2/data-structures/map)
+
+**type**
+
+The type of imap persistence, currently only supports `hdfs`.
```

Comment:
I think we need let uses know If they have no hdfs cluster, they can use `hdfs` options too.